docker/pydron.py

```diff
@@ -143,6 +90,8 @@ def do_GET(self):
     server = HTTPServer(('', PORT_NUMBER), WebHandler)
     server.serve_forever()
 
+except KeyboardInterrupt:
```

Comment:
what's this for, ooi?

Reply:
This (empirically) captures when we ask docker to stop half way through a job.